imap/imapd.c

```diff
@@ -302,7 +302,7 @@ struct capa_struct {
 static struct capa_struct base_capabilities[] = {
 /* pre-auth capabilities */
     { "IMAP4rev1",             3 },
-    { "LITERAL+",              3 }, /* This is in RFC 7888, but likely the implementation is for RFC 2088 */
+    { "LITERAL+",              3 }, /* RFC 7888 */
```

Comment:
I have a real vague memory that there was some detail of `LITERAL+` and/or `LITERAL-` that we were cheating at, which was acceptable with the older RFC but not strictly compliant with the newer one.  But I don't remember what that detail was, and maybe we already fixed it up and handle it properly now, but just neglected to update this comment at the time.  Do you know?